malariagen_data/veff.py

```diff
@@ -533,7 +496,7 @@ def _get_within_intron_effect(base_effect, intron):
     ref = base_effect.ref
     alt = base_effect.alt
     intron_start, intron_stop = intron
-    strand = base_effect.gene_strand
```

Comment:
Caught a bug here, we weren't actually populating this field, so was None. Have renamed to "strand" for simplicity and made sure we populate it above.